<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>curricula_form</class>
 <widget class="QDialog" name="curricula_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>710</y>
     <width>470</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_45">
    <item>
     <widget class="QPushButton" name="button_cancel_curricula">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_show_lingue">
      <property name="text">
       <string>Lingue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_show_attivita">
      <property name="text">
       <string>Attivita</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_generate_curricula">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_save_curricula">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>621</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Funzione</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="funzione"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_46">
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>Esperienza</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="esperienza"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLabel" name="label_49">
        <property name="text">
         <string>Sintesi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="sintesi"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_45">
        <property name="text">
         <string>Formazione</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="formazione"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLabel" name="label_48">
        <property name="text">
         <string>Competenze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="competenze"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_55"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
